std/special/panic.zig

```diff
@@ -9,8 +9,9 @@ const std = @import("std");
 pub fn panic(msg: []const u8, error_return_trace: ?*builtin.StackTrace) noreturn {
     @setCold(true);
     switch (builtin.os) {
-        // TODO: fix panic in zen.
-        builtin.Os.freestanding, builtin.Os.zen => {
+        // TODO: fix panic in zen
+        // TODO: fix panic in wasi
+        builtin.Os.freestanding, builtin.Os.zen, builtin.Os.wasi => {
```

Comment:
A bare minimum default panic handler would be `exit(1)`. Better yet would be if it could print `msg` to some kind of standard error. And then finally a pretty decent one would be one that calls `abort()` rather than `exit` (which I see is possible through the signal raising API of WASI)

Reply:
I will implement this.